src/main/java/uwuke/input/Advisor.java

```diff
@@ -0,0 +1,26 @@
+package uwuke.input;
+
+/**
+ * Advises users on potential solutions when no matching command is found.
+ * 
+ * TODO: Add more functionality to the class in the future, advise is not that helpful currently.
+ */
+public class Advisor {
+    /**
+     * Advise users on potential fixes
+     * 
+     * @param input faulty input string given by user
+     * @return advise based on faulty input
+     */
+    public static String advise(String input) {
```

Comment:
Perhaps here you could make the name of the function more indicative of what it is doing, for example - `adviseUsers`

Reply:
Will do that in the next version! It does indeed appear that this entire class is a bit ambiguous and can be named to reflect its purpose better.